dart/dynamics/Entity.h

```diff
@@ -63,11 +63,23 @@ class Shape;
 /// may have different policies about how/if their reference frame or name
 /// can be changed. Use the Detachable class to create an Entity whose reference
 /// Frame can be changed arbitrarily.
-class Entity
+class Entity : public common::Subscription
 {
 public:
   friend class Frame;
 
+  enum Notification {
```

Comment:
You're one is correct. `sendNotification(Entity::TRANSFORM)` will not work since the enum name is missing.